### PR TITLE
chore: improving doc blocks

### PR DESCRIPTION
**Pull request by vishwarajanand**
Adds documentation to some methods for clarity.

* ainit_document_table
* ainit_vectorstore_table

### CODE REVIEW

File: src/langchain_google_alloydb_pg/engine.py [hunk 1]
```diff
@@ -276,6 +276,28 @@ async def ainit_vectorstore_table(
...
+                Default: "embedding".
+            metadata_columns (List[Column]): A list of Columns to create for custom
+                metadata. Default: []. Optional.
+            metadata_json_column (str): The column to store extra metadata in JSON format.
+                Default: "langchain_metadata". Optional.
+            id_column (str):  Name of the column to store ids.
+                Default: "langchain_id". Optional,
+            overwrite_existing (bool): Whether to drop the existing table.
+                Default: False.
+            store_metadata (bool): Whether to store metadata in the table.
```
Comment (averikitsch) [inline src/langchain_google_alloydb_pg/engine.py:298]:
```suggestion
            store_metadata (bool): Whether to store metadata in a JSON column if not specified by `metadata_columns`.
```

File: src/langchain_google_alloydb_pg/engine.py [hunk 2]
```diff
@@ -276,6 +276,28 @@ async def ainit_vectorstore_table(
...
+                Default: "page_content".
+            embedding_column (str) : Name of the column to store vector embeddings.
+                Default: "embedding".
+            metadata_columns (List[Column]): A list of Columns to create for custom
+                metadata. Default: []. Optional.
+            metadata_json_column (str): The column to store extra metadata in JSON format.
+                Default: "langchain_metadata". Optional.
+            id_column (str):  Name of the column to store ids.
+                Default: "langchain_id". Optional,
+            overwrite_existing (bool): Whether to drop the existing table.
```
Comment (averikitsch) [inline src/langchain_google_alloydb_pg/engine.py:296]:
```suggestion
            overwrite_existing (bool): Whether to drop the existing table before insertion.
```

File: src/langchain_google_alloydb_pg/engine.py [hunk 3]
```diff
@@ -346,11 +368,16 @@ async def ainit_document_table(
     ) -> None:
         """
         Create a table for saving of langchain documents.
+        If table already exists, a TABLE_ALREADY_EXISTS error is thrown.
```
Comment (averikitsch) [inline src/langchain_google_alloydb_pg/engine.py:371]: Add a "Raises:" field after Args. Is the error formatted like "TABLE_ALREADY_EXIST" or is it an OperationalError with message the table already exists?

**Review by averikitsch — CHANGES_REQUESTED** [review]

File: src/langchain_google_alloydb_pg/engine.py (hunk 3 above)
Comment (vishwarajanand) [reply src/langchain_google_alloydb_pg/engine.py:371]: i will rename this to `DuplicateTableError`, becuase the error is actually like below:
```
sqlalchemy.exc.ProgrammingError: (sqlalchemy.dialects.postgresql.asyncpg.ProgrammingError) <class 'asyncpg.exceptions.DuplicateTableError'>: relation <table_name> already exists
[SQL: CREATE TABLE <full command>;]
```

```suggestion
        If table already exists, a DuplicateTableError error is thrown.
```

Comment (vishwarajanand) [reply src/langchain_google_alloydb_pg/engine.py]: Added raises as well.

**Review by averikitsch — APPROVED** [review]

**Pull request merged**